### ISSUE TITLE
Support for Fluid Fuels / Liquid Firebox and misc Questions/Feedback

### ISSUE THREAD

**Issue by Vyraal1** [elytra/BetterBoilers#11]
Better Boilers 1.1.31
Minecraft 1.12.2
Forge 2594

Just wanted to make a suggestion for possible fluid support for various liquids (excluding lava for balance reasons), such as TE's Fluids, BC Petroleum, and even Immersive Engineering Biodiesel. 

By default configs, the scale of the mod seems to fit Immersive Technology rather than Thermal Expansion/Extreme Reactors, I'd recommend reconsidering this as Immersive Technology just has steam configurations that are (on default configs) just absurd compared to other tech mods (Turbine makes 63k rf/t while on 50mb/t).  The boiler with max 

Does the boiler account for 'excessive' fireboxes in any way beyond 67 (the number at which on default configs the boiler completes one operation per tick)?

**Comment by LemmaEOF** [comment]
Fluid fuel is a very, _very_ difficult problem. I've actually made a pull request to Minecraft Forge itself to try and fix it. The primary issue is that fluids do not actually have an innate fuel value in and of themselves, so I would need to program in the exact fuel output for _every_ fluid that would be compatible. I'm working on it, but there isn't much I can do right now.

I'm not going to rebalance my mod against Immersive Tech. I'm working on the standard of 1 mB = 2 RF that every mod except Immersive Tech uses. Railcraft uses it, TE uses it, etc. The turbine I'm working on right now will function with this 1 mB to 2 RF standard as well, and I don't have plans to make that configurable. Immersive Tech's balance isn't my problem, even though I'm not too happy with it.

The boiler does count for oddities with fireboxes; instead of resetting to 0 every time it reaches 200 like a typical machine does, it decrements by 200 instead, so that it works through all the points without voiding any.

**Comment by Vyraal1** [comment]
**Fluid Fuels**: That does sound pretty clunky, nevermind, didn't realize individual fluid support for all fluids is a giant ask.

**Default Configs:** So I think I stated it unclearly, but I wasn't actually addressing the 2 mb = 1 rf conversion (I only stated that to accentuate how off-base Immersive Tech is from the norm), I'm pointing out to the fact that your current boiler configs by default are so low that it isn't **usable** by any mod at the moment **except Immersive Tech** (which we both already agree that it's way off-balance). A boiler would require over 67 Firebox Casings to make 50mb/t of steam per tick. A 2nd tier/Hardened Steam Dynamo from Thermal Expansion requires 300 mb/t of steam per tick to operate. Even old Railcraft LP/HP boilers had 360/720 mb/t steam output, which is several scales larger than the default configs.

Pack devs can easily change this, so it's a partially moot point, but just wanted to let you know that your current configs might need adjusting to fit in with the norm on steam production.

**Fireboxes**: Perfect! I like this implementation a lot then. then its scalable and rewards the user for faster fuel production.
